code/modules/mob/living/simple_animal/abnormality/aleph/melting_love.dm

```diff
@@ -36,77 +37,97 @@
 	work_damage_amount = 14
 	work_damage_type = BLACK_DAMAGE
 	/* Sounds */
-	projectilesound = 'sound/effects/attackblob.ogg'
-	attack_sound = 'sound/effects/attackblob.ogg'
-	deathsound = 'sound/effects/blobattack.ogg'
+	projectilesound = 'sound/abnormalities/meltinglove/ranged.ogg'
+	attack_sound = 'sound/abnormalities/meltinglove/attack.ogg'
+	deathsound = 'sound/abnormalities/meltinglove/death.ogg'
 	/*Vars and others */
 	loot = list(/obj/item/reagent_containers/glass/bucket/melting)
 	del_on_death = FALSE
-	var/mob/living/carbon/human/gifted_human = null
-	var/sanityheal_cooldown = 15 SECONDS
-	var/sanityheal_cooldown_base = 15 SECONDS
 	ego_list = list(/datum/ego_datum/weapon/adoration, /datum/ego_datum/armor/adoration)
 	gift_type =  /datum/ego_gifts/adoration
 	abnormality_origin = ABNORMALITY_ORIGIN_LOBOTOMY
 
+	var/mob/living/carbon/human/gifted_human = null
+	/// When FALSE - cannot attack or move
+	var/can_act = TRUE
+	/// Amount of BLACK damage done to all enemies around main target on melee attack. Also includes original target
+	var/radius_damage = 30
+
+	var/sanityheal_cooldown = 15 SECONDS
+	var/sanityheal_cooldown_base = 15 SECONDS
+
 /mob/living/simple_animal/hostile/abnormality/melting_love/Life()
 	. = ..()
 	if(gifted_human)
 		sanityheal()
 
-/mob/living/simple_animal/hostile/abnormality/melting_love/BreachEffect(mob/living/carbon/human/user)
-	..()
-	icon = 'ModularTegustation/Teguicons/96x96.dmi'
-	icon_living = "melting_breach"
-	icon_state = icon_living
-	icon_dead = "melting_breach_dead"
-	pixel_x = -32
-	base_pixel_x = -32
-	desc = "A pink hunched creature with long arms, there are also visible bones coming from insides of the slime."
-	return
-
-
 /mob/living/simple_animal/hostile/abnormality/melting_love/death(gibbed)
 	density = FALSE
-	animate(src, alpha = 0, time = 10 SECONDS)
-	QDEL_IN(src, 10 SECONDS)
-	..()
+	animate(src, alpha = 0, time = (5 SECONDS))
+	QDEL_IN(src, (5 SECONDS))
+	return ..()
 
 /* Attacks */
 /mob/living/simple_animal/hostile/abnormality/melting_love/CanAttack(atom/the_target)
 	if(isliving(the_target) && !ishuman(the_target))
 		var/mob/living/L = the_target
 		if(L.stat == DEAD)
 			return FALSE
+		if(L.type == /mob/living/simple_animal/hostile/slime && health <= maxHealth * 0.8) // We need healing
+			return TRUE
 	return ..()
 
-/mob/living/simple_animal/hostile/abnormality/melting_love/MoveToTarget(list/possible_targets)
-	if(ranged_cooldown <= world.time)
-		OpenFire(target)
+/mob/living/simple_animal/hostile/abnormality/melting_love/OpenFire(atom/A)
+	if(get_dist(src, A) < 2) // We can't fire normal ranged attack that close
```

Comment:
Doesn't Minimum Distance do this and make them back up to use it?

Reply:
What..?

Reply:
It does not do that, so nevermind.